src/Agent.Plugins/Artifact/FileContainerProvider.cs

```diff
@@ -190,6 +206,34 @@ private async Task DownloadFileContainerAsync(ArtifactDownloadParameters downloa
             }
         }
 
+        private async Task<IEnumerable<FileContainerItem>> GetArtifactItems(ArtifactDownloadParameters downloadParameters, BuildArtifact buildArtifact)
```

Comment:
Could you please add some quick comments for new methods?

Reply:
Added comments